### ISSUE TITLE
Thank you so much for your plug-in, I 'm just wondering if it can support dragging cards into a note one by one? 

### ISSUE THREAD

**Issue by tociyou1995** [slnsys/obsidian-canvas2document#6]
Thank you so much for your plug-in, which helped me convert my codified cards into notes. But with my personal depth of use, I've come up with some new functional ideas, and I don't know if that's achievable for you.   
Scenario: My usage is that I usually put cards on canvas containing my fragmentary ideas as I read, and then I review, trim, revise, and combine the cards periodically so that the necessary cards are retained, sorted by topic, and the descriptions on the cards are more accurate. The purpose of these actions is to organize my content according to the theme when I write in the note later, and to easily put the original card content directly into the note. Finally, I just need to organize the logic of the cards under each topic and polish the language a little to form a logical article or paragraph. This is great for academic writing!
Problem: The plugin now supports importing all cards from canvas into notes, but it doesn't support dragging cards into notes one by one. If I can drag a card into a note at a time, and the card moved to note disappears simultaneously in canvas, I believe this will help me achieve the above workflow better, creating more smoothly.

Thanks again for your plugin,  it really helps me a lot!

**Comment by slnsys** [comment]
I'm not sure if I understand your use case correctly, but a few points:

- Dragging a card and deleting it after dropping it into a note would be an adjustment to the interaction of the canvas, which is not easily possible for add-ons.
- And what about simply dropping a note into a document with the structural link to this card?

Two things perhaps as a workaround for your workflow?:
- partially selecting the cards in question and then converting them using Canvas2Document
- converting cards into files (Canvas function). These are then stored in the same directory as Canvas (depending on the setting) and would therefore be a Markdown document

**Comment by tociyou1995** [comment]
@slnsys 
Thank you for your thoughtful response!

When I tried the solution you provided, I found that Canvas2Document does not seem to have the functionality corresponding to the first step, which is

```partially selecting the cards in question and then converting them using Canvas2Document```

I selected part of the cards on a canvas, and applied the command STEP1 and STEP2, but I was just to obtain a document about the entire canvas (means all the card were included).
Is there something wrong with my way to use the add-on? If it already has such a function to operate on partial cards, that would be great! It would work perfectly for my workflow! :D :D 👯‍♂️

**Comment by tociyou1995** [comment]
@slnsys 
sorry, I just saw this on your wiki "There is not yet an accessible canvas api, so we cannot read the selection of nodes and other elements in the canvas. In order to convert only one subset of elements, simply copy them to another canvas and use it to convert it.
".

**Comment by slnsys** [comment]
Sorry, my second point was a bit lost in translation. I meant, what happens to the structure of the node if I drag it individually into a document? It would be removed from its links.

Regarding the conversion of selected nodes: the official API DOC still doesn't contain anything about this, but there are instructions on how to do it, and the Canvas API can actually be queried to find out what the selected nodes are. This planned feature could soon be included in canvas2document. I need to clarify whether the structure within the selected nodes is retained when queried.

**Comment by tociyou1995** [comment]
@slnsys 
Yes, so I think this command(action) should be only supported when there are no links related to the selected nodes. 
```
Sorry, my second point was a bit lost in translation. I meant, what happens to the structure of the node if I drag it individually into a document? It would be removed from its links.
```

Looking forward to this BIG up-comming feature! Thanks for your devoting again! 🎉
```
Regarding the conversion of selected nodes: the official API DOC still doesn't contain anything about this, but there are instructions on how to do it, and the Canvas API can actually be queried to find out what the selected nodes are. This planned feature could soon be included in canvas2document. I need to clarify whether the structure within the selected nodes is retained when queried.
```
